### PR TITLE
Fix(UI): Fixed policy implementation issues with team details page

### PR DESCRIPTION
**Pull request by aniketkatkar97**
### Describe your changes :

I worked on policy implementation issues with team details page

#
### Type of change :

- [x] Bug fix
- [x] Improvement

#
### Frontend Preview (Screenshots) :
<p align="center">For frontend related change, please link screenshots of your changes preview! Optional for backend related changes.
</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers

Frontend: @open-metadata/ui

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/TeamDetails/RolesAndPoliciesList.tsx [hunk 1]
```diff
@@ -29,10 +30,12 @@ const ListEntities = ({
   list,
   type,
   onDelete,
+  disableRemoveBtn,
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/TeamDetails/RolesAndPoliciesList.tsx:33]:
```suggestion
  hasAccess,
```

File: openmetadata-ui/src/main/resources/ui/src/components/TeamDetails/TeamDetailsV1.tsx [hunk 2]
```diff
@@ -874,6 +874,7 @@ const TeamDetailsV1 = ({
                     Add Role
                   </ButtonAntd>
                   <ListEntities
+                    disableRemoveBtn={!entityPermissions.EditAll}
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/TeamDetails/TeamDetailsV1.tsx:877]: Can we not use negative conditions here?
Directly pass the value and check it in the child component.

File: openmetadata-ui/src/main/resources/ui/src/components/TeamDetails/TeamDetailsV1.tsx [hunk 3]
```diff
@@ -904,6 +905,7 @@ const TeamDetailsV1 = ({
                     Add Policy
                   </ButtonAntd>
                   <ListEntities
+                    disableRemoveBtn={!entityPermissions.EditAll}
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/TeamDetails/TeamDetailsV1.tsx:908]: Same here.

File: openmetadata-ui/src/main/resources/ui/src/components/TeamDetails/RolesAndPoliciesList.tsx (hunk 1 above)
Comment (aniketkatkar97) [reply openmetadata-ui/src/main/resources/ui/src/components/TeamDetails/RolesAndPoliciesList.tsx]: Made the suggested changes

**Review by Sachin-chaurasiya — APPROVED** [review]

File: openmetadata-ui/src/main/resources/ui/src/components/TeamDetails/TeamDetailsV1.tsx (hunk 2 above)
Comment (aniketkatkar97) [reply openmetadata-ui/src/main/resources/ui/src/components/TeamDetails/TeamDetailsV1.tsx]: Made the suggested changes

File: openmetadata-ui/src/main/resources/ui/src/components/TeamDetails/TeamDetailsV1.tsx (hunk 3 above)
Comment (aniketkatkar97) [reply openmetadata-ui/src/main/resources/ui/src/components/TeamDetails/TeamDetailsV1.tsx]: Made the suggested changes

**Comment by ShaileshParmar11** [comment]
edit icon for ower is not visible can you check? due to this cypress is failing

**Pull request merged**